### PR TITLE
Add some more unit tests.

### PR DESCRIPTION
**Pull request by jawalonoski**
Adds a unit test on `App.java`, adds a disabled unit test on `Graphviz.java`, and a handful of logic tests.

### CODE REVIEW

**Comment by jawalonoski** [comment]
Huh... it looks like one build passed and the other failed. Can't figure out why that assertion is failing on Travis but not locally...

**Comment by dehall** [comment]
Have you tried restarting the failed build? I've noticed our tests can be kind of flaky sometimes and fail once but pass the next 10 times.

**Comment by jawalonoski** [comment]
Manual merge with master builds and tests successfully locally.

File: src/main/java/Graphviz.java [hunk 1]
```diff
@@ -37,29 +39,39 @@
...
+   * @param args Optional path of modules to render. If not provided,
+   *   the default modules will be loaded using the ClassLoader.
+   * @throws URISyntaxException on failure to load modules.
    */
-  public static void main(String[] args) {
+  public static void main(String[] args) throws URISyntaxException {
     File folder = Exporter.getOutputFolder("graphviz", null);
 
+    Path inputPath = null;
+    if (args != null && args.length > 0) {
```
Comment (dehall) [inline src/main/java/Graphviz.java]: I like this idea of passing in an argument to specify where to load the modules from, especially if we can eventually extend it to only load 1 module at a time, but:
1. It's not clear to me how to call it outside of an IDE. Gradle needs more config to pass command line arguments to a task, for example see the "Params" thing on the run task
2. I'm not sure what format to use to pass in a folder in Windows. I tried both "C:\folder1\etc" and "/c/folder1/etc" and neither worked.
I'd suggest using this approach which should work just as well for Mac paths and accept Windows-style paths as well:
```
      File file = new File(args[0]);
      inputPath = file.toPath();
```

Comment (jawalonoski) [reply src/main/java/Graphviz.java]: Regarding 1, it is difficult. I didn't modify the gradle task, figuring we could do it later if when we wanted to enable that. I was running it through the Eclipse run configuration, but the principle should be the same as the `run_synthea` script.

Regarding 2, it is a trying to create a URI, so `file://...` So, for example, this worked for me:

```
/Users/jwalonoski/git/synthea/src/main/resources/modules/medications
```

Regardless, I'll make the suggested change.

**Pull request merged**